packages/core/database/lib/entity-manager/index.js

```diff
@@ -611,8 +611,7 @@ const createEntityManager = (db) => {
               .whereIn(inverseJoinColumn.name, relIdsToadd)
               .where(joinTable.on || {})
               .groupBy(inverseJoinColumn.name)
-              .from(joinTable.name)
-              .transacting(trx);
```

Comment:
We are removing it because the cockroach doesn't like the select sharing the transaction with the insert. 

See: https://github.com/cockroachdb/cockroach/issues/98034

Removing it shouldn't cause any problems because putting a select inside a transaction is meaningless because nothing changes if you roll back a select.

Reply:
@Boegie19 Mentioned that this could be caused by cockroach removing the additional inserts because the `max` value is changing. Cockroach could be just doing something smart and preventing this race condition.